src/components/shared/Select/index.tsx

```diff
@@ -329,6 +339,13 @@ function Select(props: Props): React.ReactElement {
   };
   return (
     <SelectContainer>
+      <Text
```

Comment:
In this code, you use Text component of Root part together, but it is better to set default value by creating separate Title component.
And when the title is changed to optional, the title area should be exposed only when there is a title value.
```suggestion
      {title && <Title ... />}
```

Reply:
Sorry for my sooo late reply!
I've updated my commit to do what you asked.
And thanks a lot for your suggestion :)